_posts/2022-12-19-changelog-185.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #185"
+date: 2022-12-19 13:00:00 +0300
+---
+
+## New Features
+
+* [#9803] Show [type hints](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-reference-info.html#inlay-hints) inside attribute procedural macros
+
+  <img src="/assets/posts/changelog-185/inlay_type_hint_in_rocket.png" width="350px"/>
+
+* [#8544] Provide initial type inference support for [GATs](https://blog.rust-lang.org/2022/10/28/gats-stabilization.html)
+
+* [#9858] Annotate half-open [range patterns](https://doc.rust-lang.org/reference/patterns.html#range-patterns) as experimental before Rust 1.66, and highlight syntactically invalid range patterns and inclusive ranges without end as errors
+
+* [#9836] Respect [`clippy::wrong_self_convention`](https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention) attribute, avoiding the need to use additional [IntelliJ-specific](https://jetbrains.com/help/idea/disabling-and-enabling-inspections.html#suppress-inspections) `//noinspection RsSelfConvention` comments anymore (by [@Techcable])
+
+* [#9816] Add support for the "Foreign items may not have type/const parameters" ([E0044](https://doc.rust-lang.org/error_codes/E0044.html)) error (by [@kuksag])
```

Comment:
```suggestion
* [#9816] Detect `Foreign items may not have type/const parameters` ([E0044](https://doc.rust-lang.org/error_codes/E0044.html)) error
```

I believe we shouldn't mention our interns in the same way as external contributors. I will update the script which generates changelog template